snomed/com.b2international.snowowl.validation.snomed/src/tests/java/com/b2international/snowowl/validation/snomed/GenericValidationRuleTest.java

```diff
@@ -113,4 +113,45 @@ public void ruleSnomedCommon2() throws Exception {
 				ComponentIdentifier.of(SnomedTerminologyComponentConstants.DESCRIPTION_NUMBER, d2.getId()));
 	}
 	
-}
+	@Test
+	public void ruleSnomedCommon3() throws Exception {
+		final String ruleId = "snomed-common-3";
+		indexRule(ruleId);
+
+		// Relationships with deprecated characteristic types
+		SnomedRelationshipIndexEntry relationshipWithDefiningCharType = relationship(generateConceptId(), Concepts.IS_A, generateConceptId())
```

Comment:
Add another relationship with a type outside of the characteristic type hierarchy. 
The rule should NOT report that as an error.